test/test_nn.py

```diff
@@ -1800,6 +1800,10 @@ def add_test(test):
         test_cuda=False
     ),
     dict(
+        constructor=lambda: nn.CosineSimilarity(),
```

Comment:
I'm sorry but I havent found any way to include two parameters at `forward`. Until then I cant merge the thing since the test will fail. :(